macos_instructions.yml

```diff
@@ -0,0 +1,132 @@
+    - text:  https://developer.apple.com/forums/thread/672087?answerId=659036022#659036022
+    - code:
+        run: git || true
+        working-directory: dockstore
+    - text: _(If that doesn't work install git manually https://git-scm.com/download/mac)_
+    - newLine: 2
+    - text: Setup git user information
+    - code:
+        run: git config --global user.email "you@example.com";git config --global user.name "Your Name" # ; will be replaced with new lines in README.md
+        working-directory: dockstore
+    - text: |
+        [Read about git token requirements](https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/)
+    - text: |
+        [Setup personal access token for git CLI](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+    - text: |
+       [It's helpful to cache your git personal access token](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git)
+    - text: Install Hubflow
+    - text: https://datasift.github.io/gitflow/TheHubFlowTools.html
+    - code:
+        run: git clone https://github.com/datasift/gitflow;cd gitflow;sudo ./install.sh # ; will be replaced with new lines in README.md
+        working-directory: dockstore
+    - newLine: 1
+    - text: Install JDK 17
+    - text: https://formulae.brew.sh/formula/openjdk@17
+    - code:
+        run: brew install openjdk@17
+        working-directory: dockstore
+    - text: Download and install node.js
+    - text: https://github.com/nvm-sh/nvm
+    - code:
+        run: curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+        working-directory: dockstore
+    - text: Install git secrets
+    - text: https://github.com/awslabs/git-secrets
+    - code:
+        run: brew install git-secrets
+        working-directory: dockstore
+    - text: Install wget
+    - code:
+        run: brew install wget
+        working-directory: dockstore
+    - newLine: 1
+    - text: Install jq
+    - code:
+        run: brew install jq
+        working-directory: dockstore
+    - text: '#### Build the webservice'
+    - text: (cd to where you cloned the dockstore/dockstore repo)
+    - code:
+        run: ./mvnw clean install
+        working-directory: dockstore
+    - newLine: 1
+    - text: '#### Build the UI'
+    - text: (cd to where you cloned the dockstore/dockstore-ui2 repo)
+    - newLine: 1
+    - text: Set up UI requirements
+    - text: 'NOTE: You must use the --legacy-peer-deps switch due to using npm version 8.11.0 (> npm 6) '
+    - text: for reasons mentioned in [this post](https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh)
+    - code:
+        run: npm ci --legacy-peer-deps
+        working-directory: dockstore-ui2
+    - text: Run prebuild
+    - code:
+        run: npm run prebuild
```

Comment:
This is not necessary. `npm run build` on line 123 already executes the `prebuild` step.

Reply:
I removed the prebuild step from both the action and `README.md`.

[f7083ae](https://github.com/dockstore/dockstore/pull/5043/commits/f7083ae00d76337feb82fcdf03a0b602f9908681)